client/lib/presales-chat/index.ts

```diff
@@ -40,8 +36,7 @@ function getGroupName( keyType: KeyType ) {
 
 export function usePresalesChat( keyType: KeyType, enabled = true, skipAvailabilityCheck = false ) {
 	const isEnglishLocale = useIsEnglishLocale();
-	const { canConnectToZendesk } = useChatStatus();
-	const isEligibleForPresalesChat = enabled && isEnglishLocale && canConnectToZendesk;
+	const isEligibleForPresalesChat = enabled && isEnglishLocale;
```

Comment:
I'd advise against this change. This was actually added in another PR related to this project: https://github.com/Automattic/wp-calypso/pull/79679 😅 

`canConnectToZendesk` is a cheap and fast check that tells us accurately if the user will be able to load the Zendesk's scripts or not (based on the security settings of their browser). There's no point in any other checks if this one returns false. Removing this check will not make Zendesk load magically - if the check failed, the script will not load anyway, so there will be no Zendesk launcher, etc.

If anything, I'd use the value of `canConnectToZendesk` to offer some other, backup support to the user. Either mentioning they need to adjust their browser's settings (but that's a bit meh and not exactly sets them up for best sales experience) or just show alternative resources that might help them in lieu of the pre-sales chat (I haven't checked the flow, so don't know if there's already something there or not).

Reply:
Thanks. I wonder if the issue with the other PR then is that `canConnectToZendesk` returns false on loading.

https://github.com/Automattic/wp-calypso/blob/trunk/packages/help-center/src/hooks/use-chat-status.ts#L31